conf/app.ini

```diff
@@ -551,3 +551,12 @@ SHOW_FOOTER_BRANDING = false
 SHOW_FOOTER_VERSION = true
 ; Show time of template execution in the footer
 SHOW_FOOTER_TEMPLATE_LOAD_TIME = true
+
+[markup.asciidoc]
+ENABLED = true
```

Comment:
I think that this example markup renderer should be disabled by default or in this example because it requires third party tools.

Reply:
@JonasFranzDEV this file is an example file. It will not effect anything if you upgrade.

Reply:
The example should follow the default values, shouldn't it?

Reply:
`[markup.xxxx]` are dynamic sections. Default values is no `[markup.xxxx]` section as this PR did. I put this section on the example ini file for others know how to add one or more tools. Also I will add these to config cheatsheet but I think put it in example ini file is easier to find for users.